.stylelintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": ["@wagtail/stylelint-config-wagtail"],
+  "rules": {
+    "scale-unlimited/declaration-strict-value": [
+      ["/color/", "fill", "stroke"],
+      {
+        "ignoreValues": [
```

Comment:
I’m surprised you chose JSON for this file! Not a blocker but it’d be nice to be able to comment on why specific rules are overridden like this.

Reply:
Yeah I just went for the simplest way to get the config going